create_deposition/README.md

```diff
@@ -0,0 +1,42 @@
+# Action for Creating Zenodo-Depostion
+This action creates a deposition with the metadata provided in `metadata.json` file.
+
+## Usage
+To create a deposition,first add the metadata to `metadata.json` file in the root directory. 
+Run `ci/create_new_depostion.workflow`.
+
+```hcl
+workflow "Create Zenodo Depostion" {
+  resolves = "create deposition"
+}
+action "create deposition" {
+  uses = "create_deposition/"
```

Comment:
the reference will be to this repository, so something like

```hcl
uses= "popperized/zenodo/create_deposition@master"
```

We could rename this action to `create`, since IMO that's less verbose and we always create depositions, i.e. there are no other things on Zenodo that get created. but you can leave it like `create_deposition`  if you wish